pkg/lambdaworks/lambdaworks_test.go

```diff
@@ -427,6 +427,40 @@ func TestToU64Fail(t *testing.T) {
 	}
 }
 
+func TestToU321(t *testing.T) {
+	felt := lambdaworks.FeltOne()
+	result, err := felt.ToU32()
+
+	var expected uint32 = 1
+
+	if expected != result {
+		t.Errorf("Error in conversion expected: %v, got %v with err: %v", expected, result, err)
+	}
+
+}
+
+func TestToU3210230(t *testing.T) {
+	felt := lambdaworks.FeltFromUint64(10230)
+	result, err := felt.ToU32()
+
+	var expected uint32 = 10230
+
+	if expected != result {
+		t.Errorf("Error in conversion expected: %v, got %v with err: %v", expected, result, err)
+	}
+}
```

Comment:
Maybe separate the the number from the type in the name with an underscore, or put the number first, so it's easier to read as two separate entities.